### DISCUSSION TITLE
Freac frequently freezing

### DISCUSSION THREAD

**Discussion by Harpper** [enzo1982/freac#583] [Help & Support]
I opened a bug report on this but got little feedback or response. I have a fairly high spec PC runing Linux Mint 21.2 (Ryzen 7600X CPU 32GB RAM, PCIE SSD etc). Why is Freac 1.17 freezing so frequently but not with every FLAC rip? 2 cores appear to max out at 100% until Freac freezes and I have to do a force quit. Settings are full paranoia mode and verify input other than that it is a pretty default set up. Several versions of Freac ago I don't think this happened, and it does not happen with Asunder which I may be forced to

**Comment by djfake** [comment]
I have the same issue. It had worked flawlessly for years, but now can barely get one disc ripped. Will try a new disc drive.

**Comment by Harpper** [comment]
My thought is it is possibly due to cdparanoia being enabled. I will try with a brand new disc and see what happens. There should be a way to prevent FREAC from using 100% of one or two cores but leave the other cores virtually unused. It seems poor allocation of the CPUs

EDIT: Checked a disk without cdparanoia and the program still maxes out 2 cores on my PC

**Reply by joshstoik1** [reply]
Try unchecking "Enable parallel processing" under the "Resources" settings. That seemed to fix the issue for me.

**Reply by Harpper** [reply]
Thanks for the suggesation, it works great. I am not sure if that setting was on when I installed or I enableed it.It is a relief to get FREAC working properly

**Reply by djfake** [reply]
Thanks, appears to be working for me as well with parallel processing off. Still have issues with discs, so will try replacing the drive.
